scripts/spec_xml/generate_spec_xml.py

```diff
@@ -29,21 +29,34 @@
     os.path.join(os.path.dirname(__file__), '..', '..'))
 DEFAULT_OUTPUT_DIR_1_3 = os.path.abspath(
     os.path.join(DEFAULT_CHIP_ROOT, 'data_model', '1.3'))
+DEFAULT_OUTPUT_DIR_IN_PROGRESS = os.path.abspath(
+    os.path.join(DEFAULT_CHIP_ROOT, 'data_model', 'in_progress'))
 DEFAULT_OUTPUT_DIR_TOT = os.path.abspath(
     os.path.join(DEFAULT_CHIP_ROOT, 'data_model', 'master'))
 DEFAULT_DOCUMENTATION_FILE = os.path.abspath(
     os.path.join(DEFAULT_CHIP_ROOT, 'docs', 'spec_clusters.md'))
 
+# questions
+# is energy-calendar still in?
+# is heat-pump out? wasn't in 0.7
+# location-cluster - is this define gone now?
+# queuedpreset - is this define gone now?
+CURRENT_IN_PROGRESS_DEFINES = ['aliro', 'atomicwrites', 'atomicwrites', 'device-location', 'e2e', 'energy-calendar', 'energy-drlc',
+                               'energy-management', 'hrap-1', 'hvac', 'matter-fabric-synchronization', 'metering', 'secondary-net',
+                               'solar-power', 'tcp', 'water-heater', 'wifiSetup']
```

Comment:
When I compiled the necessary claims review, I used the following in progress defines: 

```
aliro atomicwrites battery-storage device-location e2e-jf energy-calendar energy-drlc energy-management heat-pump hrap-1 hvac matter-fabric-synchronization metering secondary-net service-area-cluster solar-power tcp water-heater wifiSetup
```

To that end, we should be able to replicate this via
```suggestion
CURRENT_IN_PROGRESS_DEFINES = ['aliro', 'atomicwrites', 'battery-storage', 'device-location', 'e2e-jf', 'energy-calendar', 'energy-drlc',
                               'energy-management', 'heat-pump', 'hrap-1', 'hvac', 'matter-fabric-synchronization', 'metering', 'secondary-net',
                               'service-area-cluster', 'solar-power', 'tcp', 'water-heater', 'wifiSetup']
```

Reply:
perfect - thank you. I will add those, re-scrape and resolve the todos above. New commit incoming this afternoon.